napari/benchmarks/benchmark_shapes_layer.py

```diff
@@ -27,48 +27,164 @@
     from napari.benchmarks.utils import Skip
 
 
-class Shapes2DSuite:
+class BackendType(Enum):
+    bermuda = auto()  # data processing using bermuda, (https://github.com/napari/bermuda), preprocessing using numba
+    partsegcore = auto()  # data preprocessing using PartSegCore (https://partseg.github.io), edge triangulation using numba
+    triangle = auto()  # data preprocessing using numba, edge triangulation using numba, triangulation using triangle
+    numba = auto()  # data preprocessing and edge triangulation using numba, triangulation using vispy
+    pure_python = auto()  # data preprocessing, edge triangulation, and triangulation using python and vispy
+
+    def __str__(self):
+        return self.name
+
+    def __repr__(self):
+        return self.name
+
+
+backend_list = list(BackendType)
```

Comment:
```suggestion
backends = list(BackendType)
```

Changing name to backends would improve readability. There is no need to say that it is a list.